locale/lang-ca.js

```diff
@@ -2723,7 +2723,23 @@ SnapTranslator.dict.ca = {
         'oposat',
     '__shout__go__':
         'bandera verda premuda',
-    'new costume %l width %dim height %dim':
-        'nou vestit %l amplada %dim alçada %dim'
+    'download script':
+        'descarrega el bloc de programa',
```

Comment:
Should be "descarrega el programa". This option downloads the whole script, not just a block.